streaming.go

```diff
@@ -93,6 +93,7 @@ func (sp *streamProcessor) startOnce(ch chan<- bool) {
 				sp.setInitializedOnce.Do(func() {
 					sp.config.Logger.Printf("Started LaunchDarkly streaming client")
 					sp.isInitialized = true
+					ch <- true
```

Comment:
When the initial payload of features is too big and we have to manually fetch it we were failing to properly signal that we had started up